HW_3/main.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask
+from models import db, User
+from flask_wtf.csrf import CSRFProtect
+from flask import render_template, request
+from form import LoginForm
+
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///mydatabase.db'
+db.init_app(app)
+app.config['SECRET_KEY'] = 'secret_key'
+csrf = CSRFProtect(app)
+
+
+@app.route('/', methods=['GET', 'POST'])
```

Comment:
Если используете объект по классу `CSRFProtect`, то нужно добавлять `@csrf.exempt`
```suggestion
csrf = CSRFProtect(app)


@app.route('/', methods=['GET', 'POST'])
@csrf.exempt
```